docs/user_v2/tutorials/installation.rst

```diff
@@ -1,3 +1,8 @@
+.. note::
```

Comment:
I've noticed that the v2 docs suggest installing via pip, the v1 docs suggest installing via conda through the nsls2forge channel, and that I don't use either of these in favor of installing from conda-forge. I'm assuming that we should either be suggesting the pip or conda-forge approaches everywhere.

Reply:
Personally, I've never used conda, so I only put in the install instructions from PyPI using pip. The install plan at DLS is via PyPI, so I'd prefer to only write about what I know. Could we make PyPI the primary source then someone could add a How-To guide for installing from conda as an extra step?

Reply:
I think pypi as the primary source makes a lot of sense and a conda install section can be added later. This only stood out to me because the v1 and v2 docs suggested different paths.